src/catalog/schema_utils.py

```diff
@@ -24,15 +24,15 @@
 from src.utils.logging_manager import LoggingManager
 
 
-class Utils(object):
+class SchemaUtils(object):
```

Comment:
It's not necessary here. But in general, we can create a class of static methods if there can be multiple implementations of the method. For code readability, it would help to keep it in a class.